src/signalo/core/migrations/0001_initial.py

```diff
@@ -29,7 +46,7 @@ class Migration(migrations.Migration):
                 (
                     "geom",
                     django.contrib.gis.db.models.fields.PointField(
-                        srid=4326, verbose_name="Geometry"
+                        srid=2056, verbose_name="Geometry"
```

Comment:
we cannot have harcoded SRID in the migration, we need to sort this out at some point

Reply:
Well how hard is it hardcoded? Isn't a different SRID just one migration away?